seed/models/data_quality.py

```diff
@@ -129,31 +135,36 @@ class Rule(models.Model):
             'not_null': True,
             'rule_type': RULE_TYPE_DEFAULT,
             'severity': SEVERITY_ERROR,
+            'condition': RULE_INCLUDE,
```

Comment:
'not_null' in this context came from the original 'not null' check column which is different from the newly defined 'not null' and 'required'. Rule.not_null is to check if the values exists, which disables the min/max column (here for address_line_1 would be 'text must contain'). Since by default, all DEFAULT_RULES check for valid values (valid address line 1 that contains a user defined string). That said, default rules should all check the validity of a field (include, exclude, range), not existence (required, not null). Line 'not_null': True should be removed (leads back to my remove filed question this morning).

Reply:
Are you saying the Rule.not_null attribute was only being used to disable the min/max column on the frontend? 

My impression is that the effective purpose of this current implementation (of `condition='not null'`) and the previous implementation (`Rule.not_null = True`) should remain the same - to identify when the value for this field is null in the DB for any particular record. Am I misunderstanding the purpose either before or after the changes we have in this PR?

Reply:
Rule.not_null and Rule.required are used to: 1. disable range values on the front end and ;2. filter out null values in the model as error fields so that detailed error message only report 'null' error once instead of multiple times in case it falls in other rules for the same field. Previous 'not_null' is equivalent to now 'range', 'include' and 'exclude'.

Reply:
I'm a little fuzzy on this still. Are you saying that, before, rule.not_null and rule.required were just fallbacks for whenever other criteria like min and max checks or "contain/does not contain" checks were not met? Sorry, I'm not quite sure what you meant by item 2. 

I don't know how the previous set up of rule.not_null being True or False is equivalent to now `condition` being set to "range", "include", or "exclude".